index.bs

```diff
@@ -1965,6 +1965,38 @@ See also:
 
 * <a href="https://www.w3.org/2001/tag/doc/promises-guide">Writing Promise-Using Specifications</a>
 
+<h4 id="sync-callbacks">When to use callback functions instead</h4>
+
+<p>If an algorithm requires that callback functions respond
+synchronously (e.g. to take or veto some action), then that is a
+synchronous API, and should not be modelled with a promise. Instead,
+use an explicit callback or event.</p>
+
+<p>In Promise reaction callbacks, you may not require action within
```

Comment:
It'd be nice to link this so people can get a firm definition of what it is. Perhaps:

```suggestion
<p>In [Promise reaction callbacks](https://tc39.es/ecma262/multipage/control-abstraction-objects.html#sec-promisereaction-records),
you may not require action within
```